test/unit/auditor_test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+let chai = require("chai");
+let sinon = require("sinon");
+let sinonChai = require("sinon-chai");
+let expect = chai.expect;
+chai.use(sinonChai);
+
+import auditor from "../../src/auditor";
+
+describe("default", () => {
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'describe' is not defined.